core/domain/interaction_registry.py

```diff
@@ -105,3 +109,41 @@ def get_all_specs(cls):
             interaction.id: interaction.to_dict()
             for interaction in cls.get_all_interactions()
         }
+
+    @classmethod
+    def get_all_specs_for_state_schema_version(cls, state_schema_version):
+        """Returns a dict containing the full specs of each interaction for the
+        given state schema version, if available.
+
+        Args:
+            state_schema_version: int. The state schema version to retrieve
+                interaction specs for.
+
+        Returns:
+            dict. The full specs of each interaction for the given state schema
+            version. A mapping of interaction id to the specs of the
```

Comment:
The first two sentences of the description seem to overlap, perhaps combine them into one. (Also, the second "sentence" is really a sentence fragment...)